### DISCUSSION TITLE
Snyk support once again?

### DISCUSSION THREAD

**Discussion by anon-pr** [badges/shields#10316] [Q&A] [answered]
Hi there,

I remember using shields.io for my older project in the past. Now I've revisited it once again because I cannot locate where to obtain Snyk markdown badge from their dashboard, is there even a way? So i thought I would visit shields.io because there are tons of badges here, but I couldn't locate one for Snyk.io, is it discontinued? Is there feature plans for it? 

From this issue: #9349 , I've gathered it is deprecated now 

Thanks

**Comment by skywarth** [comment]
Following this thread

**Comment by chris48s** [comment, marked as answer]
We removed the snyk badges because it stopped working when the endpoint we were using to get the number of vulnerabilities for a repo was removed.

In principle, we could re-add it if there is somewhere we could get the data from. I'm not sure if anyone looked into whether it would be viable to do it via the Snyk API. If the info is available and someone wants to submit a PR, we could review it.
